admin_manual/configuration_server/antivirus_configuration.rst

```diff
@@ -203,9 +203,66 @@ interactive ClamAV scanning command. ownCloud should find it automatically.
 When you are satisfied with how ClamAV is operating, you might want to go
 back and change all of your logging to less verbose levels.
 
+Rule Configuration
+^^^^^^^^^^^^^^^^^^
+
+ownCloud provides the abilility to customise how ownCloud reacts to the
```

Comment:
For some reason neither VIM nor Atom would spell-check the file when it was identified as a reStructuredText file. Still trying to find out why that's happening.